### PR TITLE
chore: add test for container image

### PR DESCRIPTION
**Pull request by alexeagle**
### Type of change

- Chore (any other change that doesn't affect source or test files, such as configuration)

### Test plan

- New test cases added

### CODE REVIEW

**Review by thesayyn — APPROVED** [review]

File: oci_python_image/hello_world/test.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# See https://github.com/GoogleContainerTools/container-structure-test#command-tests
+schemaVersion: 2.0.0
+commandTests:
+  - name: 'run'
+    command: /opt/hello_world/hello_world_bin
```
Comment (f0rmiga) [inline oci_python_image/hello_world/test.yaml:5]: Can you omit this? We want the test to assert is the default entrypoint is correct.

**Review by f0rmiga — CHANGES_REQUESTED** [review]

**Comment by f0rmiga** [comment]
Of course it was merged before my review.

File: oci_python_image/hello_world/test.yaml (hunk 1 above)
Comment (alexeagle) [reply oci_python_image/hello_world/test.yaml:5]: No, you can't:
https://github.com/GoogleContainerTools/container-structure-test#command-tests

but, we can add a metadataTest which checks the entrypoint, will add in a follow-up

Comment (f0rmiga) [reply oci_python_image/hello_world/test.yaml:5]: Interesting. Thanks for pointing it out.

**Pull request merged**